### PR TITLE
Add a 4th KYC signer.

### PR DESCRIPTION
**Pull request by Tritium-VLK**
One of the old ones left.

### CODE REVIEW

**Review by gosuto-inzasheru — APPROVED** [review]

File: extras/signers.json [hunk 1]
```diff
@@ -63,7 +63,8 @@
     "Arbitrum": {
       "kycsigner1": "0x5c43d19ee1b9f93143c7c258501ef3ada1204524",
```
Comment (Tritium-VLK) [inline extras/signers.json:64]:
```suggestion
      "kycsigner1_retired": "0x5c43d19ee1b9f93143c7c258501ef3ada1204524",
```

Comment (Tritium-VLK) [reply extras/signers.json:64]: Identified retired signer

**Comment by Tritium-VLK** [comment]
Signer swap KYC1 -> KYC4

**Comment by Tritium-VLK** [comment]
https://app.onchainden.com/safes/arb1:0xb6BfF54589f269E248f99D5956f1fDD5b014D50e/transactions/0x916cbe9bd4ddc443f045cf684e002c9771260c5f38b1c28b2f54da1ce734f4f7?nid=316181

**Pull request merged**